controlplane/kubeadm/api/v1alpha4/kubeadm_control_plane_webhook_test.go

```diff
@@ -268,6 +274,10 @@ func TestKubeadmControlPlaneValidateUpdate(t *testing.T) {
 						},
 					},
 				},
+				NTP: &bootstrapv1.NTP{
+					Servers: []string{"test-server-1", "test-server-2"},
+					Enabled: enabledNTP,
```

Comment:
```suggestion
					Enabled: pointer.BoolPtr(true),
```
I think that's easier to digest :)

Reply:
Done :)